common/components/BalanceSidebar/AccountInfo.tsx

```diff
@@ -46,23 +49,53 @@ export default class AccountInfo extends React.Component<Props, State> {
     });
   };
 
+  public displayHWAddress = () => {
+    const { hwType, wallet } = this.props;
+    const { dPath, index } = wallet as any;
+    if (hwType === 'ledger') {
+      ledger.comm_u2f.create_async().then(comm => {
+        new ledger.eth(comm).getAddress_async(dPath + '/' + index, true, false);
+      });
+    } else if (hwType === 'trezor') {
```

Comment:
Confirming this works when `hwType === 'tezor'`. Other than the typo, showed up fine on my device.